### ISSUE TITLE
Install eXpand with install folder in parameter

### ISSUE THREAD

**Issue by isatahiri** [eXpandFramework/XpandPosh#2]
Hello Tolis,

In our company the GPO have changed. No more executable in "My Documents" is it possible de define the installation folder of expand  when we run the install from powershell ?

BR
ISA

**Comment by isatahiri** [comment]
So now when i run the command : 
Set-ExecutionPolicy Bypass -Scope Process -Force;iex "$(([System.Net.WebClient]::new()).DownloadString('http://install.expandframework.com'));Install-Xpand -Assets @('Assemblies','Nuget','VSIX','Source')  #-Version '18.2.704.0'"

![image](https://user-images.githubusercontent.com/41955268/56971570-c1487480-6b69-11e9-8a07-fd7031a81fba.png)

BR
ISA

**Comment by apobekiaris** [comment]
do u mean that this script works with no issues

```ps1
Set-ExecutionPolicy Bypass -Scope Process -Force;iex "$(([System.Net.WebClient]::new()).DownloadString('https://raw.githubusercontent.com/eXpandFramework/XpandPosh/master/XpandPosh/Public/Xpand/Install-Xpand.ps1'));Install-Xpand -Assets @('Assemblies','Nuget','VSIX','Source') #-Version '18.2.704.0'"
```

**Comment by apobekiaris** [comment]
i only change the invoker last change on the script was 28 days ago

https://github.com/eXpandFramework/XpandPosh/commits/master/XpandPosh/Private/Xpand/InstallXpand.ps1

**Comment by isatahiri** [comment]
The script works without any issue on my personnal dev computer. 
The issue is in my side. I cannot run it anymore in the company's computer since new GPO are implemented and blocking exe file being ran in MyDocuments folder. 

So when the script runs the nuget.exe in MyDocuments\eXpandFramework, then it fails. 

I guess my workaround in to do the installation manually like before or change the Documents\eXpandFramework\InstallXpand.ps1 and change the variable $InstallationPath

Would that work ?

**Comment by apobekiaris** [comment]
yes u can change the installation path, or if that not work I could add a paramter to use nuget from $env:Path. Let me know

**Comment by apobekiaris** [comment]
maybe gpo does not allow changing environmental vars as well?

**Comment by apobekiaris** [comment]
Chocolatey sounds a more transparent option

does gop allows to install this in your system?

```ps1
Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
```
